crates/ruff_python_formatter/tests/snapshots/format@statement__match.py.snap

```diff
@@ -456,13 +452,38 @@ match foo:
         # leading
         # leading
         # leading
-        "NOT_YET_IMPLEMENTED_PatternMatchValue",  # trailing
+        "a",  # trailing
         # trailing
         # trailing
         # trailing
-        "NOT_YET_IMPLEMENTED_PatternMatchValue",
+        "b",
     ]:
         pass
+
+
+match foo:
+    case 1:
+        y = 0
+    case ((1)):
```

Comment:
What would the formatting be like after the fix for double parentheses? Would it still keep the parentheses from the original code i.e., format is as `(1)`?

Reply:
Ah nevermind, it's answered in your other PR.